amazon-link-engine/trunk/amazon-link-engine.php

```diff
@@ -147,7 +149,7 @@ function genius_ale() {
 }
 
 if (!is_admin()) {
-  add_action('wp_head', 'genius_ale');
+  add_action('wp_head', 'genius_ale', 9999);
```

Comment:
Great, that might actually solve a bug someone reported last week where JQuery was not defined (despite WP ostensibly having built-in JQuery?). I'll give this a test.

Thank you for contributing.